README.md

```diff
@@ -256,6 +256,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [First Citdael](http://www.skyislandsvr.com/pages/firstcitadel.html) - Citadel with flowing waterfalls
 - [Missed Connections](http://armthethinker.github.io/webVR-experiments/5--missed-connections.html) - Visualization of Craigslist "Missed Connections" section
 - [Aeon](http://aeon.horoman.com/) (by [@h0r0man]) - Aeon is a 3D environment with VR support, where the user runs an island that tells about the prequel to the main story of Aeon.
+- [SlideAmaze](http://slideamaze.com/) - Turns your slideshow presentation [into a 3D maze](http://fouryears.eu/2016/07/27/slide-a-maze/).
```

Comment:
this doesn't block getting merged at all, but, just as an aside, you should probably look into supporting `https`. if you aren't able to easily enable it on your server, or use [Let's Encrypt](https://letsencrypt.org/) or [CloudFlare](https://www.cloudflare.com/), totally understood. just thought I'd mention it.